.github/workflows/test.yaml

```diff
@@ -0,0 +1,41 @@
+name: Test container image build
+
+on:
+  pull_request_target:
+    types: [labeled, synchronize]
+    paths:
+      - '.github/**/*'
+      - 'conda/**'
+    branches:
+      - main
```

Comment:
Hi @scottyhq, we're trying to test out this docker container build in a Pull Request, but the GitHub Actions doesn't seem to be running. Do you think there's something in the repo permissions that needs to change? I've copied this `pull_request_target` event from https://github.com/ICESAT-2HackWeek/website2022/blob/10229090a94edb3e5734dbc00149b0502ad5396a/.github/workflows/test.yaml#L8-L18 and looked at https://github.blog/2020-08-03-github-actions-improvements-for-fork-and-pull-request-workflows, but still can't figure out why the CI is silent.

Reply:
`pull_request_target` will require that the workflow needs to be in the `main` branch before it'll run for subsequent PRs. It's a somewhat confusing setup I admit, but my suggestion is merge just the github actions first, then follow up with creating the environment. 

You might consider following the instructions to use the hackweek template repository (https://github.com/uwhackweek/jupyterbook-template) But adds complexity by adding the event website! Previously we used a specific image building template that could use updating, but maybe that would also simplify things for this use-case https://github.com/uwhackweek/docker-template

Reply:
> `pull_request_target` will require that the workflow needs to be in the `main` branch before it'll run for subsequent PRs. It's a somewhat confusing setup I admit, but my suggestion is merge just the github actions first, then follow up with creating the environment.

Ah I see. I did try using just the `pull_request` event but it doesn't seem to work for forks. The merge first, update environment later seems like a good approach. Let me remove the `build.yaml` first since the quay.io environment isn't set up yet.

> You might consider following the instructions to use the hackweek template repository (https://github.com/uwhackweek/jupyterbook-template) But adds complexity by adding the event website! Previously we used a specific image building template that could use updating, but maybe that would also simplify things for this use-case https://github.com/uwhackweek/docker-template

Yep, I've cherry-picked parts of the hackweek template and 2i2c template at https://github.com/2i2c-org/hub-user-image-template. After chatting with Tasha just now, I think we agreed to keep the repo2docker setup within this 'website' repo for now, rather than in a dedicated repo like 'cryo-image'. Could always split it out later I suppose.